modules/platform/nuxeo-fsexporter/src/test/java/org/nuxeo/io/fsexporter/test/TestFSExporterCollection.java

```diff
@@ -0,0 +1,102 @@
+package org.nuxeo.io.fsexporter.test;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.Serializable;
+
+import javax.inject.Inject;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.nuxeo.common.Environment;
+import org.nuxeo.ecm.collections.api.CollectionManager;
+import org.nuxeo.ecm.collections.core.test.CollectionFeature;
+import org.nuxeo.ecm.core.api.Blob;
+import org.nuxeo.ecm.core.api.CoreSession;
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.core.api.PathRef;
+import org.nuxeo.ecm.core.api.impl.blob.StringBlob;
+import org.nuxeo.io.fsexporter.FSExporter;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+
+@RunWith(FeaturesRunner.class)
+@Features(CollectionFeature.class)
+@Deploy("nuxeo-fsexporter")
+public class TestFSExporterCollection {
+
+    @Inject
+    CoreSession session;
+
+    @Inject
+    CollectionManager collectionManager;
+
+    @Inject
+    FSExporter service;
+
+    @Test
+    public void ShouldExportCollection() throws Exception {
+
+        String COLLECTION_NAME = "CollectionNameTest";
+        String COLLECTION_DESCRIPTION = "dummy";
```

Comment:
According to the [java convention naming](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html), names such as these ones are for constants and not for inside method variable.